packages/nextjs/hooks/scaffold-eth/useTransactor.tsx

```diff
@@ -55,15 +56,17 @@ export const useTransactor = (_signer?: Signer, gasPrice?: number): TTransaction
       } else {
         throw new Error("Incorrect transaction passed to transactor");
       }
-      toast.remove(toastId);
+      hotToast.remove(toastId);
```

Comment:
If we are using a wrapper for hotToast, we should try not using it directly.

What about creating a toast.remove inside our custom toast object?

```ts
  remove: (toastId: string) => {
    hotToast.remove(toastId);
  }
```